tests/test_elements.py

```diff
@@ -128,51 +129,73 @@ def test_ST_Equal_Column_WKTElement(self, geometry_table):
         }
 
 
-class TestWKBElement():
+class TestExtendedWKBElement():
+
+    # _bin/_hex computed by following query:
+    # SELECT ST_GeomFromEWKT('SRID=3;POINT(1 2)');
+    _bin = buffer_(b'\x01\x01\x00\x00 \x03\x00\x00\x00\x00\x00\x00'
+                   b'\x00\x00\x00\xf0?\x00\x00\x00\x00\x00\x00\x00@')
+    _hex = str_('010100002003000000000000000000f03f0000000000000040')
+    _srid = 3  # expected srid
+    _wkt = 'POINT (1 2)'  # expected wkt
 
     def test_desc(self):
-        e = WKBElement(b'\x01\x02', extended=True)
-        assert e.desc == '0102'
+        e = WKBElement(self._bin, extended=True)
```

Comment:
Previously test data where `bytes`, but we assume Postgis only returns `buffer`.

Reply:
Right.